scripts/levitate-show-affected-plugins.js

```diff
@@ -124,7 +123,7 @@ function printAffectedPluginsSection(data) {
       const affectedPlugins = getColumn(rows, pluginsColumnIndex);
 
       markdown += `<h3>Number of affected plugins: ${affectedPlugins.size}</h3>`;
-      markdown += "<p>To check the plugins affected by each import, click on the links below.</p>";
+      markdown += '<p>To check the plugins affected by each import, click on the links below.</p>';
```

Comment:
This change appeared after merging main. I cannot undo it as it changes each time I commit the change.